src/main/java/org/folio/entlinks/service/authority/AuthorityServiceI.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.entlinks.service.authority;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import org.folio.entlinks.domain.entity.Authority;
+import org.springframework.data.domain.Page;
+import org.springframework.transaction.annotation.Transactional;
+
+public interface AuthorityServiceI<T> {
```

Comment:
maybe it can be named ```IAuthorityService```

Reply:
Checkstyle will produce an error for IAuthorityService